sagemaker-debugger/build_your_own_container_with_debugger/debugger_byoc.ipynb

```diff
@@ -0,0 +1,904 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "from smdebug.core.modes import ModeKeys\n",
+    "\n",
+    "# Retrieve the loss tensors collected in training mode\n",
+    "y = []\n",
+    "for step in trial.tensor('loss').steps(mode=ModeKeys.TRAIN):\n",
+    "    y.append(trial.tensor('loss').value(step,mode=ModeKeys.TRAIN)[0])\n",
+    "y=np.asarray(y)\n",
+    "\n",
+    "# Retrieve the loss tensors collected in evaluation mode\n",
+    "y_val=[]\n",
+    "for step in trial.tensor('loss').steps(mode=ModeKeys.EVAL):\n",
+    "    y_val.append(trial.tensor('loss').value(step,mode=ModeKeys.EVAL)[0])\n",
+    "y_val=np.asarray(y_val)\n",
+    "\n",
+    "train_save_points=int(trial.tensor('scalar/train_steps_per_epoch').value(0)[0]/train_save_interval)\n",
+    "val_save_points=int(trial.tensor('scalar/valid_steps_per_epoch').value(0)[0]/eval_save_interval)\n",
+    "\n",
+    "y_mean=[]\n",
+    "x_epoch=[]\n",
+    "for e in range(int(trial.tensor('scalar/epoch').value(0)[0])):\n",
```

Comment:
nit: save `trial.tensor('scalar/epoch').value(0)[0]` as it is used a couple of times